docs: first qualification draft

OscarAblinger opened the pull request:
PR for #5

OscarAblinger committed:
docs: first qualification draft

AdrienLemaire commented on Project qualifications.md:
```diff
@@ -0,0 +1,91 @@
+### 1. The project must accept contributions
+
+This is rather obvious:
+If it isn't accepting code from other users, there is no
```
If it's obvious, do we really need to write about it?

OscarAblinger replied:
Well, my thought process was to add it, since it is a requirement for the project – an obvious one, true, but it still might be nice to write it down in a document?
If you feel like it's useless, I don't mind removing it.

AdrienLemaire commented on Project qualifications.md:
```diff
@@ -0,0 +1,91 @@
+itself.
+
+Any form of racism, sexism or other kind of hostility will
+disqualify a project from this list.
```
I'm not sure how we'll be able to evaluate that.
Do this kind of OSS project even exist?

OscarAblinger replied:
I'd hope they are rare. I didn't plan on trying to evaluate every project based on this, but rather just have it in case we need it.
I thought, that for any project to be on a list for potential contributors, they should also have a welcoming atmosphere.

I, honestly, don't have much experience in contributing to OSS, so, if you think, it's a useless point, I'll remove it.

AdrienLemaire commented on Project qualifications.md:
```diff
@@ -0,0 +1,91 @@
+
+### 4. The project must treat contributors with respect
+
+OS contributors engage in good faith and in their free time.
```
I'd usually write OSS (Open-Source Software), to remove any confusion with OS (Operating System).

AdrienLemaire commented on Project qualifications.md:
```diff
@@ -0,0 +1,91 @@
+### 1. There are no license restrictions
+
+Any license is fair game, as long as it allows for contributions
+by other individuals.
```
Any Open-source license is fine, ie it should be OSI approved:
https://opensource.org/licenses/alphabetical

AdrienLemaire commented on Project qualifications.md:
```diff
@@ -0,0 +1,91 @@
+
+There are no locality restrictions on the projects.
+They can operate globally, on a national level or only in a single
+city.
```
That's kind of a weird statement for a project called "awesome-global-contributions" though :P 
What I meant during our conversation is that it doesn't matter if the project community isn't English-speaking, but if the project doesn't have a "potential" global impact, maybe we shouldn't list it?

AdrienLemaire reviewed (commented):
Thanks for putting this document together! I wouldn't have been able to write it better, and maybe somebody else can write an even better version later on. 
Maybe we can add a badge "draft" next to the title to highlight that this document is open for improvements?

OscarAblinger committed:
docs: incorporate some resolved PR-changes

As of PR #9, the following resolved changes were made:

- OS was changed to OSS to avoid confusion with OS (Operating System)
- License restriction was updated to require OSI approved licenses

OscarAblinger commented:
> I wouldn't have been able to write it better

I feel like that's not true, but thank you ^^

> Maybe we can add a badge "draft" next to the title to highlight that this document is open for improvements?

Sure, makes sense :)

AdrienLemaire committed:
chore: rename file + add draft badge

AdrienLemaire commented:
@AblingerOscar I took the liberty to add the badge and rename the file.
I suggest we add one file per project in the `src/` directory. Github-wise, having 1 PR per project could go smoothly, avoiding any kind of conflict, and leaving us time to spend more or less time reviewing projects .

AdrienLemaire reviewed (approved):
LGTM, thank you!
